src/include/job.h

```diff
@@ -891,6 +891,7 @@ task_find	(job		*pjob,
 #define JOB_SVFLG_ChkptMig 0x100 /* job has migratable checkpoint */
 #define JOB_SVFLG_Suspend  0x200 /* job suspended (signal suspend) */
 #define JOB_SVFLG_StagedIn 0x400 /* job has files that have been staged in */
+#define JOB_SVFLG_HASHOLD  0x800 /* job has a hold request sent to MoM */
```

Comment:
I'm surprised that you actually found a void in the job flags :)

Reply:
Yup!  Good idea.